test/Crossroads.Test.Linux/Services/UnitTest1.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace Crossroads.Test.Linux.Services
+{
+    public class UnitTest1
```

Comment:
This looks like leftover, please remove. It's not even using the new attribute.

Reply:
Yes, it doesn't make use of the custom attribute, was like a placeholder therefore it will be removed